src/permutations.rs

```diff
@@ -36,19 +36,20 @@ impl<F: Field> MonomialPermutation<F> {
 
 impl<F: Field> Permutation<F> for MonomialPermutation<F> {
     fn permute(&self, builder: &mut GadgetBuilder<F>, x: &Expression<F>) -> Expression<F> {
-        builder.exp(x, &self.n)
+        builder.exponentiation(x, &self.n)
```

Comment:
Renamed to `exponentiation` for notation consistency away from references to `Expression`.